code/API_definitions/network_access_management.yaml

```
@@ -0,0 +1,2863 @@
+openapi: 3.0.3
+
+info:
+  title: CPE Management API
+  version: 0.0.1

```

Version should be "wip" until the commit is part of a tagged (pre-)release. See https://wiki.camaraproject.org/display/CAM/API+versioning

```suggestion
  version: wip

```

---

Thanks @hdamker for taking a look!  @RandyLevensalor and I did previously discuss anything that wasn't a simple patch would be spun off into its own issue/PR.